### ISSUE TITLE
Xcode 9 build errors

### ISSUE THREAD

**Issue by agisboye** [nolanw/HTMLReader#80]
Xcode 9 has some new defaults with regards to warnings. When building from the HTMLReader.xcodeproj itself, this is not an issue because the new "recommended project settings" have not been enabled for the project. When building with Carthage, however, said project settings are enabled and cause the build to fail.

```
HTMLSelector.m:403:61: warning: implicit conversion from nullable pointer
'HTMLSelectorPredicateGen _Nullable' (aka 'BOOL (^)(HTMLElement * _Nonnull __strong)') to non-nullable pointer type 'ObjectType _Nonnull' (aka 'id')
```

**Comment by nolanw** [comment]
Thanks for the reminder! This was fixed in 808c2ebedc8b044eefc7a46c2dfca593244922e1 but it's well past time I push it out in a release. HTMLReader 2.0.6 should build cleanly for you on Xcode 9.

Please holler back if I did something silly and it's still broken for you!

**Comment by agisboye** [comment]
Works like a charm. Cheers. 😃
